### DISCUSSION TITLE
백준 2304 창고 다각형 문제 추가 요청드립니다.

### DISCUSSION THREAD

**Discussion by yeonjungin** [tony9402/baekjoon#46] [문제 추가/수정 요청]
[필수] 문제 이름 : 창고 다각형
[필수] 문제 번호 : 2304
[필수] 문제 링크 : https://www.acmicpc.net/problem/2304
[필수] Tag: 스택, 완전탐색

**Comment by tony9402** [comment]
문제 검토 결과 완전탐색(Brute Force) 태그에 추가하였습니다.
https://github.com/tony9402/baekjoon/tree/main/brute_force
